Cargo.toml

```diff
@@ -28,6 +28,7 @@ combine = "2.5"
 libc = "0.2"
 time = "0.1"
 byteorder = "1.2"
+hex = "0.4.3"
```

Comment:
This is required for the plugin which is under `examples/`, so it should probably be in `dev-dependencies` instead?